python-nectar.spec

```diff
@@ -1,9 +1,11 @@
 %{!?python_sitelib: %global python_sitelib %(%{__python} -c "from %distutils.sysconfig import get_python_lib; print get_python_lib()")}
+%{!?python3_sitelib: %global python3_sitelib %(%{__python3} -c "from %distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%global sum A download library that separates workflow from implementation details
```

Comment:
i follow this instruction https://fedoraproject.org/wiki/Packaging:Python
show me other and i will change :)